### ISSUE TITLE
Discrepancy with milestone due date

### ISSUE THREAD

**Issue by adamreisnz** [ZenHubIO/support#1303]
I created a new milestone, selected the start date as November 20th, yet after saving it showed "Due today" and upon checking in edit milestone, the due date was set as November 19th. Looks likely to be a timezone issue (I'm in NZDT).

**Comment by kendramoroz** [comment]
Hey @adamreisnz thanks for reporting! We're looking into a few discrepencies with Milestones at the moment. I'll keep you posted on updates with this Issue!

**Comment by schierle** [comment]
I am having the same issue: in Zenhub web app, I set the milestone due date to, say, May 12th, but it gets created with May 11th. I'm in UTC+2 (CEST).

Any updates?

**Comment by kendramoroz** [comment]
👋 Hey everyone - we updated the display of dates across the app. Can everyone confirm if this still an issue?

**Comment by adamreisnz** [comment]
Thanks for fixing!

First thing I noticed is that creating a new milestone has the same problem as with #1437

I created a new milestone, and it doesn't show up in the filter dropdown. Even after I assign an issue to it, it still doesn't appear in the milestone dropdown. Only after a refresh of the page does it appear.

The date issue appears to be resolved though, but I'll have to do more testing with at different times of the day.
